### PR TITLE
Omit platform-specific openers from coverage.

### PR DESCRIPTION
**Pull request by theacodes**

### CODE REVIEW

File: .coveragerc [hunk 1]
```diff
@@ -1,5 +1,9 @@
 [report]
-omit = */samples/*
+omit =
+    */samples/*
+    # Don't report coverage over platform-specific modules.
+    oauth2client/contrib/_fcntl_opener.py
```
Comment (dhermes) [inline .coveragerc:5]: @jonparrott Couldn't we test this one on Travis?

Comment (theacodes) [inline .coveragerc:5]: Yes? Do you want to? As soon as multistore file has 100% coverage, I'm swapping out all of this for fasteners.

Comment (dhermes) [inline .coveragerc:5]: Ah OK. No worries then. OK to merge? (Also do you have a timeline for `multistore` getting 100% and when you're swapping out for fasteners?)

Comment (theacodes) [inline .coveragerc:5]: You can merge if you're satisfied.

Timeline for multistore is today, as I'm working on it right now.

Timeline for fasteners is likely a week or so off.

**Pull request merged**